synapse/groups/attestations.py

```diff
@@ -166,7 +168,7 @@ async def on_renew_attestation(
 
         return {}
 
-    def _start_renew_attestations(self) -> None:
+    def _start_renew_attestations(self) -> Deferred[None]:
```

Comment:
I think you need quotes for old-Twisted.

```suggestion
    def _start_renew_attestations(self) -> "Deferred[None]":
```

Reply:
Thanks, I didn't realise that that was the cause of the slew of duplicate-metric CI test failures.